amida-tech/orange-api
# added creator field to doses and journal entries

This adds a `creator` field to doses and journal entries, so that the client can display the name of whoever created a dose of entry.

The `creator` field is populated as the first and last name of the user who made the request to create the entry or dose. It is not updated when the dose or entry is updated. It is a string rather than a reference to the user document in mongo because there are no endpoints in the API to look at other users' details. If all we had was a user ID the client wouldn't be able to lookup their name, email, etc to display. Adding this functionality would be a bigger lift than the current PR. Also I don't think we want a mechanism in the API for seeing other users info.

### to test
* checkout `feature/Notes-Record-Type` for the ignite app
* create a new user
* make a note
* refresh the app
* go view the note
* make sure the users name appears at the top of the note detail view
